styles.css

```diff
@@ -414,106 +397,712 @@ body {
+    content: "";
+    width: 235.78px;
+    height: 261.79px;
+    margin-left: -470%;
+    margin-top: -300%;
+    background-image: url('img/illustration-1-contact-form-desktop.png');
+    background-size: contain;
+    background-position: left bottom;
+    background-repeat: no-repeat;
+    z-index: 3;
+  }
+
+  #six {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    content: "";
+    width: 130.28px;
+    height: 105.85px;
+    margin-left: -390%;
+    margin-top: -260%;
+    background-image: url('img/illustration-2-contact-form-desktop.png');
+    background-size: contain;
+    background-position: center center;
+    background-repeat: no-repeat;
+    z-index: 3;
+  }
+}
+
+@media (max-width: 768px) {
+  #four {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    align-content: flex-end;
+    content: "";
+    width: 313.9px;
+    height: 448.7px;
+    margin-left: 100%;
+    margin-top: -400%;
+    background-image: url('img/contact\ form-illustratoin-3-desktop.png');
+    background-size: contain;
+    background-position: center center;
+    background-repeat: no-repeat;
+    overflow: hidden;
+    z-index: 3;
+  }
+
+  #five {
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    align-content: flex-start;
+    content: "";
+    width: 235.78px;
+    height: 261.79px;
+    margin-left: 0;
+    margin-top: -300%;
+    background-image: url('img/illustration-1-contact-form-desktop.png');
+    background-size: contain;
+    background-position: left bottom;
+    background-repeat: no-repeat;
+    z-index: 3;
+  }
+
+  #six {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    content: "";
+    width: 130.28px;
+    height: 105.85px;
+    margin-left: 300%;
+    margin-top: -260%;
+    background-image: url('img/illustration-2-contact-form-desktop.png');
+    background-size: contain;
+    background-position: center center;
+    background-repeat: no-repeat;
+    z-index: 3;
+  }
+}
+
+@media only screen and (min-width: 375px) {
   body {
-    font-size: 25px;
+    background-image: url('img/header-illsutration-mobile.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    align-content: center;
+  }
+
+  .container {
+    padding-top: 50px;
+  }
+
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50px;
+    z-index: 1;
+  }
+
+  nav {
+    position: absolute;
+    top: 0;
+    left: -50px;
+    width: 100%;
+    z-index: 1;
+    display: none;
+  }
+
+  .grid-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    grid-gap: 25px 30px;
+    width: 100%;
+    margin: 0;
+  }
+}
+
+@media (max-width: 768px) {
+  div {
+    overflow-x: hidden;
```

Comment:
- [ ] Since you are designing a mobile first website, there is no need stating media queries for the mobile version. You only have to define media queries for the desktop version since all the initial styling will be for the mobile version. Even if there is the need to define for that, it should be done once and not multiple times as you have done on lines 1005 and 1103. Always remember the DRY principle. 
- [ ] Despite working on the desktop version, you should also make sure that the mobile version is maintained and matches the actual design. As shown in the image below, You design is missing the hamburger menu. The headline section also pushes into the works section at certain screen widths. Kindly check and fix this issues. :pray: 

### Your design 

![Screenshot from 2023-06-08 08-05-05](https://github.com/NgachaIan/Personal_Portfolio/assets/65233860/661c35e7-e2b4-49dd-ba19-77e878528f1f)



### Actual design 


![Screenshot from 2023-06-08 07-57-10](https://github.com/NgachaIan/Personal_Portfolio/assets/65233860/b28b406b-8f26-42ce-96b6-f7019c557444)